cfme/infrastructure/virtual_machines.py

```diff
@@ -302,12 +302,16 @@ class InfraVmReconfigureView(BaseLoggedInPage):
     cpu_total = WInput()  # read-only, TODO widgetastic
 
     disks_table = DisksTable()
+    affected_vms = WTTable(VersionPick({Version.lowest(): '//*[@id="records_div"]/table',
```

Comment:
REQUIRED : please rename web_ui widgets Table and Input instead. Those will be removed soon.

Reply:
Maybe the guy who removes `web_ui` can rename this later, because I'm not sure which tests would get affected due to this, and they pass or fail currently, it would go our of context of this PR.
I can put a #todo here for renaming